src/v/kafka/server/handlers/fetch.h

```diff
@@ -359,9 +360,11 @@ struct fetch_plan {
     explicit fetch_plan(
       size_t shards,
       op_context::latency_point start_time = op_context::latency_clock::now())
-      : fetches_per_shard(shards, shard_fetch(start_time)) {
-        for (size_t i = 0; i < fetches_per_shard.size(); i++) {
-            fetches_per_shard[i].shard = i;
+      : fetches_per_shard() {
```

Comment:
Isn't it better to add the expected ctor overload to `chunked_vector`? That's sort of been the principle I've tried to use since if once you run it it's likely to occur again the future.

Reply:
fetches_per_shard is actually not a chunked_vectorbut a normal std::vector. This just needs changing because the fetches_per_shard value type now contains a chunked_vector and hence is not copyable anymore.

Reply:
OK that explains why I couldn't find exactly where the type changed.